src/routes/contracts/__tests__/get-contract.e2e-spec.ts

```diff
@@ -31,7 +31,8 @@ describe('Get contract e2e test', () => {
       address: '0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4',
       name: 'CreateCall',
       displayName: '',
-      logoUri: null,
+      logoUri:
```

Comment:
A logo was added for this contract on the Transaction Service, breaking this test.